### PR TITLE
added desktop version

### PR DESCRIPTION
**Pull request by ElShaer398**
Adding desktop version of Portfolio-setup-and-mobile-first project at Microverse.

### CODE REVIEW

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @ElShaer398  👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request 🔴 

You have submitted a project, that does not have a pull request summary, and the project setup part of our README is missing

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: main.css [hunk 1]
```diff
@@ -447,3 +554,292 @@ form {
...
+    width: 1156px;
+    height: 496px;
+    display: grid;
+    align-items: stretch;
+    padding: 24px;
+    grid-template-columns: 544px 496px;
+    grid-template-rows: repeat(6, auto);
+    column-gap: 46px;
+    row-gap: 30px;
+  }
```
Comment (didierganthier) [inline main.css:634]: Nice job designing this card 🎨 but it's not taking the whole width of the screen for the Desktop version. See the differences between your design and Figma. 👇🏽 

## Figma

Screenshot 2023-02-25 at 23 47 07

## Your design

Screenshot 2023-02-25 at 23 47 32

File: main.css [hunk 2]
```diff
@@ -447,3 +554,292 @@ form {
...
+
+  .about-myself-section {
+    background: #fff;
+    padding: 142px;
+    display: grid;
+    grid-template-columns: 447px 684px;
+    grid-template-rows: 114px 102px 47px 47px 48px;
+    column-gap: 50px;
+    row-gap: 30px;
+  }
```
Comment (didierganthier) [inline main.css:719]: Great section 👍🏽 but look into using % to make sure that the space is attributed properly see the differences between yours and the Figma. 👇🏽 

## Figma

Screenshot 2023-02-25 at 23 49 46

## Your design

Screenshot 2023-02-25 at 23 50 10

As you can see in the Figma ☝🏽 it looks like the `About myself` part is taking `1/3` of the width and that the `Languages` part is taking `2/3` (Desktop version) 😉

File: main.css [hunk 3]
```diff
@@ -447,3 +554,292 @@ form {
   color: grey;
   border-color: grey;
 }
+
+@media (min-width: 768px) {
```
Comment (didierganthier) [inline main.css:558]: Well done on the responsiveness overall 💯 but at some point when resizing your screen there's some issue where the design doesn't take the full width of the screen. Take a look below 👇🏽 

Screenshot 2023-02-25 at 23 39 48
--------
Screenshot 2023-02-25 at 23 40 09

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @ElShaer398 ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

File: main.css (hunk 3 above)
Comment (topeogunleye) [inline main.css:558]: - [ ] I kindly suggest that you commit the changes requested by the previous reviewer:
> Well done on the responsiveness overall 💯 but at some point when resizing your screen there's some issue where the design doesn't take the full width of the screen. Take a look below 👇🏽


| Name     | Character |
| ---      | ---       |
| Your Design | image    |
| Portfolio Design    |  image |

File: main.css (hunk 2 above)
Comment (topeogunleye) [inline main.css:719]: - [ ] Please commit the changes requested by the previous reviewer:
> Great section 👍🏽 but look into using % to make sure that the space is attributed properly see the differences between yours and the Figma. 👇🏽

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @ElShaer398, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* Professional README file. :heavy_check_mark:
* PR has a good title and summary :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @ElShaer398, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* .
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**